cves/kernel/CVE-2016-6130.yml

```diff
@@ -147,11 +143,11 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
-autodiscoverable:
+  answer: From Pengfei Wang, I found this Double-Fetch bug in Linux-4.5/drivers/s390/char/sclp_ctl.c when I was examining the source code.
```

Comment:
This sentence is lacking some context. Try to explain how _Pengfei Wang_ found this vulnerability? Remember your audience might not be familiarized with the Linux Kernel

Reply:
I refactored it and I hope it's clearer now.